### ISSUE TITLE
Update to Concrete5 8.0.3

### ISSUE THREAD

**Issue by raytrax** [hissy/addon_disqus#1]
Hi,
I have tried to integrate the plugin in the latest version of concrete5 but it shows version error.
You have a plan to get an update? thank you!

**Comment by hissy** [comment]
Yes I'll update, but I have a many packages. please wait soon ;-)
